dist/tools/flake8/check.sh

```diff
@@ -9,17 +9,17 @@
 
 FLAKE8_CMD="python3 -m flake8"
 
-if tput colors &> /dev/null && [ $(tput colors) -ge 8 ]; then
-    CERROR="\e[1;31m"
-    CRESET="\e[0m"
+if tput colors &> /dev/null && [ "$(tput colors)" -ge 8 ]; then
+    CERROR=$'\033[1;31m'
+    CRESET=$'\033[0m'
 else
     CERROR=
     CRESET=
 fi
 
 DIST_TOOLS=${RIOTBASE:-.}/dist/tools
 
-. ${DIST_TOOLS}/ci/changed_files.sh
+. "${DIST_TOOLS}/ci/changed_files.sh"
```

Comment:
shellcheck still reports a warning with this line. Maybe add this comment on the line before to remove it:
```sh
# shellcheck source=/dev/null
```
See [here](https://github.com/koalaman/shellcheck/wiki/SC1090) for information

Reply:
Or use 
```sh
# shellcheck disable=SC1090
```
?

Reply:
I'd rather disable this check in a future CI script, similar to the flake8.conf. This to prevent having to add an exception to almost every script